fast64_internal/sm64/sm64_objects.py

```diff
@@ -1826,6 +1831,61 @@ def get_behavior_params(self):
         return self.bparams
 
 
+class SM64_SegmentProperties(bpy.types.PropertyGroup):
+    seg5_load_custom: bpy.props.StringProperty(name="Segment 5 Seg")
+    seg5_group_custom: bpy.props.StringProperty(name="Segment 5 Group")
+    seg6_load_custom: bpy.props.StringProperty(name="Segment 6 Seg")
+    seg6_group_custom: bpy.props.StringProperty(name="Segment 6 Group")
+    seg5_enum: bpy.props.EnumProperty(name="Segment 5 Group", default="Do Not Write", items=groupsSeg5)
+    seg6_enum: bpy.props.EnumProperty(name="Segment 6 Group", default="Do Not Write", items=groupsSeg6)
```

Comment:
```suggestion
    seg5_group: bpy.props.EnumProperty(name="Segment 5 Group", default="Do Not Write", items=groupsSeg5)
    seg6_group: bpy.props.EnumProperty(name="Segment 6 Group", default="Do Not Write", items=groupsSeg6)
```

Reply:
sorry to be picky with these names btw but since we can't easily change them, may as well be willing to improve them now

Reply:
these enums are used for both the segment load and the group that is loaded using the script, so I don't really want to call them "group"

Reply:
Colloquially speaking hackers use that name, and it is how people will select, but I use it to add the load because that is implicitly defined by the group.
I want to make it clear that when using string input they are defined separately, but when using the enum the load and group are defined.